app/assets/javascripts/repositories/row_editor.js

```diff
@@ -22,13 +22,41 @@ var RepositoryDatatableRowEditor = (function() {
   }
 
   function validateAndSubmit($table, $submitButton) {
-    let $form = $table.find(`.${EDIT_FORM_CLASS_NAME}`);
-    let $row = $form.closest('tr');
+    let $forms = $table.find(`.${EDIT_FORM_CLASS_NAME}`);
+    let $row = $forms.closest('tr');
     let valid = true;
     let directUrl = $table.data('direct-upload-url');
     let $files = $table.find('input[type=file]');
     $row.find('.has-error').removeClass('has-error').find('span').remove();
 
+    // register submits event for row forms
+    let formsCount = $forms.length;
+    $forms.each(function() {
+      const form = $(this);
+      form.off().on('submit', function(event) {
```

Comment:
any way to avoid using 'off' ?

Reply:
Not sure. We need it if we want to change data in case some forms failed in submitting.
But we actually need to unregister only the `submit` event :+1: